src/styles.js

```diff
@@ -17,10 +17,10 @@ var css = require('./styles.css');
 
 var defaultStyles = {
   background: 'rgba(61, 61, 61, 1)',
-  accentBackground: '#BA8FF7',
+  accentBackground: '#008673',
```

Comment:
`#ba8ff7` is close to *violet-30*, so think we should probably go with `#00baa1` (*teal-30*), especially since this is meant to be an accent.